Header header
# one of start, stop
string command
# full scan is the 360 turn. Non-full are for corrections while moving towards a cup
bool full